### PR TITLE
Fix scroll and arrow navigation

### LINKED ISSUE #643: Arrow keys don't move cell focus in cmd mode in the notebook
This means there's no easy way to navigate a long notebook with the keyboard.

### LINKED ISSUE #651: Scrollbar gone in the notebook
Just today I noticed that opening a long notebook now has no scroll bar, nor is it possible to scroll it at all even with the wheel/touchpad.

This is definitely a new bug for me, anyone else can reproduce it? If so, I'd say it's pretty critical.

### PR DESCRIPTION
**Pull request by blink1073**
Fixes #651, fixes #643.

### CODE REVIEW

File: src/shortcuts/plugin.ts [hunk 1]
```diff
@@ -201,16 +201,6 @@ const SHORTCUTS = [
     keys: ['B']
   },
   {
-    command: 'notebook-cells:select-previous',
```
Comment (blink1073) [inline src/shortcuts/plugin.ts]: Note: these were duplicate *and* using the wrong command name.

**Comment by jasongrout** [comment]
I think the problem with the scroll bar disappearing (#651) was that the scrolling role of the container was pushed down to the notebook widget in the changes around #415. We still see that the container has `overflow: auto` at https://github.com/jupyter/jupyterlab/pull/653/files#diff-26fc09b216b5f9869fabd7dec2427fe2R24. Also, the scroll into view functionality doesn't work because previously we had to scroll the parent, but now we should scroll the notebook element itself.

**Comment by blink1073** [comment]
Thanks, fixed!

**Comment by jasongrout** [comment]
Can you take off the `overflow: auto` for the container too?

**Comment by blink1073** [comment]
Done, and confirmed that it works as expected.

**Comment by jasongrout** [comment]
Confirmed. Thanks!

**Pull request merged**